odxtools/compumethods/compucodecompumethod.py

```diff
@@ -47,17 +47,17 @@ def __post_init__(self) -> None:
                   "CompuCodeCompuMethod must exhibit COMPUCODE category")
 
     def convert_internal_to_physical(self, internal_value: AtomicOdxType) -> AtomicOdxType:
-        odxraise(r"CompuCodeCompuMethod cannot be executed", DecodeError)
+        odxraise(r"CompuCodeCompuMethod cannot be executed by odxtools", DecodeError)
         return cast(AtomicOdxType, None)
 
     def convert_physical_to_internal(self, physical_value: AtomicOdxType) -> AtomicOdxType:
-        odxraise(r"CompuCodeCompuMethod cannot be executed", EncodeError)
+        odxraise(r"CompuCodeCompuMethod cannot be executed by odxtools", EncodeError)
         return cast(AtomicOdxType, None)
 
     def is_valid_physical_value(self, physical_value: AtomicOdxType) -> bool:
-        odxraise(r"CompuCodeCompuMethod cannot be executed", NotImplementedError)
+        # CompuCodeCompuMethod cannot be executed by odxtools
         return False
 
     def is_valid_internal_value(self, internal_value: AtomicOdxType) -> bool:
-        odxraise(r"CompuCodeCompuMethod cannot be executed", NotImplementedError)
+        # CompuCodeCompuMethod cannot be executed by odxtools
```

Comment:
What do the changes in this file have to do with the PR?

Reply:
this is in the second commit ("improve `parameter_info()`"). I basically stumbled over this and I thought that the `.is_valid_*_value()` methods should not raise exceptions even if `CompuCode` compu methods cannot be directly supported (as so oftenm I don't have any files which use this attrocity, so I'm pretty much indifferent).